tools/dev/msftidy.rb

```diff
@@ -435,6 +435,8 @@ def check_ranking
       if not available_ranks.include?($1)
         error("Invalid ranking. You have '#{$1}'")
       end
+    elsif @source =~ /['"](SideEffects|Stability|Reliability)['"]\s+=/
```

Comment:
Whitespace *should* be present, but doesn't have to be.

```suggestion
    elsif @source =~ /['"](SideEffects|Stability|Reliability)['"]\s*=/
```

Other `msftidy` checks also use `\s*` match.